superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/ag-grid/ContextMenu/MenuItems/SubmitToAssemblyLineMenuItem.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Menu } from 'src/components/Menu';
+import Icon from '@ant-design/icons';
+import AlSvg from '../../../cccs-grid/images/al.svg';
+
+interface SubmitToAssemblyLineMenuItemProps {
+  label: string;
+  data: string[];
+  onSelection: () => void;
+  key?: string;
+  disabled?: boolean;
+  isContextMenu?: boolean;
+  contextMenuY?: number;
+}
+
+export default function SubmitToAssemblyLineMenuItem(
+  props: SubmitToAssemblyLineMenuItemProps,
+) {
+  // Handler for clicking on the item
+  const onClick = () => {
+    // Loop through each item in the data array and open a new window for each URL
+    // Using encodeURIComponent to prevents URL injection issues and ensures that special characters in the data strings are correctly interpreted by the URL
+    props.data.forEach(item => {
```

Comment:
nit: how do you feel about a more descriptive variable name? Like `eml_path`?

Reply:
Fixed in new commit.